### DISCUSSION TITLE
Documentación de prueba

### DISCUSSION THREAD

**Discussion by ma-robles** [grupoioa/gis_agm_series#80] [Show and tell]
@MaCuur subir aquí los archivos de prueba. Me parece que al menos aca sí se pueden los PDF

**Comment by MaCuur** [comment]
[test01.pdf](https://github.com/grupoioa/gis_agm_series/files/8145604/test01.pdf)
[test02.pdf](https://github.com/grupoioa/gis_agm_series/files/8145605/test02.pdf)
[test06.docx](https://github.com/grupoioa/gis_agm_series/files/8145606/test06.docx)
[Documentación_webpage_01.pdf](https://github.com/grupoioa/gis_agm_series/files/8145607/Documentacion_webpage_01.pdf)
La última fue utilizando una página de internet =P

**Comment by MaCuur** [comment]
Otras versiones:

[test03.pdf](https://github.com/grupoioa/gis_agm_series/files/8165660/test03.pdf)
[test07.docx](https://github.com/grupoioa/gis_agm_series/files/8165661/test07.docx)
[test05.pdf](https://github.com/grupoioa/gis_agm_series/files/8165662/test05.pdf)
